README中文文档翻译 (#304)
microsoft/HydraLab: README.zh-CN.md

![Tech Architecture](docs/images/technical_architecture.png)

<span id="for-user"></span>
### 对于Hydra Lab的使用者:

> - [触发Hydra Lab测试服务中的测试任务运行](https://github.com/microsoft/HydraLab/wiki/Trigger-a-test-task-run-in-the-Hydra-Lab-test-service)

Reviewer: 也许可以改成这样：
"在 Hydra Lab 测试服务中触发一次测试任务"

Reviewer: ```suggestion
- [在Hydra Lab测试服务中触发一次测试任务](https://github.com/microsoft/HydraLab/wiki/Trigger-a-test-task-run-in-the-Hydra-Lab-test-service)
```

- [部署测试代理服务](https://github.com/microsoft/HydraLab/wiki/Deploy-a-test-agent-service)
- [创建Appium UI测试自动化项目](https://github.com/microsoft/HydraLab/wiki/Create-an-Appium-UI-Test-Automation-Project)

> 注意：如果您是Microsoft FTE并希望加入内部Hydra Lab测试服务，请访问 [我们的SharePoint网站](https://microsoftapc.sharepoint.com/teams/MMXDocument/SitePages/Hydra-Lab-test-automation-service-onboarding-guideline.aspx) 以了解有关内部服务实例的详细信息。

<span id="for-contributor"></span>
### 参与贡献Hydra Lab:

> - [为Hydra Lab GitHub项目贡献力量](https://github.com/microsoft/HydraLab/wiki/Contribute-to-the-Hydra-Lab-GitHub-Project)

Reviewer 2: ```suggestion
- [如何参与和贡献Hydra Lab GitHub项目？](CONTRIBUTING.md)
```

> - [如何参与并为Hydra Lab GitHub项目做出贡献？](https://github.com/microsoft/HydraLab/wiki/Contribute-to-the-Hydra-Lab-GitHub-Project)

Reviewer 2: ```suggestion
- [如何参与并为Hydra Lab GitHub项目做出贡献？](CONTRIBUTING.md)
```

<span id="who-use-it"></span>
## 谁在使用Hydra Lab?

它已经支持以下Microsoft产品的UI测试自动化：
- Microsoft Phone Link（Windows UWP应用程序）和 Link to Windows（Android应用程序）
- Microsoft Launcher (Android)
- Microsoft Outlook/Edge (Android/iOS)
- Microsoft Fluent UI Android/Yammer Android

<span id="contribute"></span>
## 贡献

> 您对Hydra Lab的贡献将使整个测试自动化生态系统发生变化。请参阅贡献指引 **[CONTRIBUTING.md](CONTRIBUTING.md)** 。

Reviewer: 也许这样更通顺一些：“你对Hydra Lab的贡献将为整个测试自动化生态系统带来改变。”

Reviewer: ```suggestion
您对Hydra Lab的贡献将为整个测试自动化生态系统带来改变。请参阅贡献指引 **[CONTRIBUTING.md](CONTRIBUTING.md)** 。
```

> ### 贡献者英雄墙：

Reviewer 2: ```suggestion
### 贡献者英雄榜：
```

<a href="https://github.com/Microsoft/hydralab/graphs/contributors">
  <img src="https://contrib.rocks/image?repo=Microsoft/hydralab" />
</a>

<span id="contact"></span>
## 联系我们

> 请尽情投入吧!如果您对Hydra Lab有任何疑问，或者您想就您遇到的问题与我们联系，您可以通过以下方式联系我们：

Reviewer 2: ```suggestion
如果您对Hydra Lab有任何疑问，您可以通过以下方式联系我们：
```

> - [打开 issue](https://github.com/microsoft/HydraLab/issues/new) 或提交 PRs.

Reviewer 2: ```suggestion
- [创建 issue](https://github.com/microsoft/HydraLab/issues/new) 或提交 PRs.
```

> - 用邮件联系我们: [hydra_lab_support@microsoft.com](mailto:hydra_lab_support@microsoft.com).

Reviewer 2: ```suggestion
- 发送邮件到: [hydra_lab_support@microsoft.com](mailto:hydra_lab_support@microsoft.com).
```

<span id="links"></span>

> ## 链接

Reviewer 2: ```suggestion
## 相关链接
```

- [Secure a Java web app using the Spring Boot Starter for Azure Active Directory.](https://docs.microsoft.com/en-us/azure/developer/java/spring-framework/configure-spring-boot-starter-java-app-with-azure-active-directory) 
- [Appium: Cross-platform automation framework for all kinds of your apps built on top of W3C WebDriver protocol.](https://github.com/appium/appium)
- [Google Android Tools Ddmlib: A ddmlib jar that provides APIs for talking with Dalvik VM.](https://android.googlesource.com/platform/tools/base/+/master/ddmlib/)

<span id="ms-give"></span>

> ## 微软赞助商

Reviewer 2: ```suggestion
## 微软Give员工捐赠活动
```

感谢您以Hydra Lab的名义为 [微软员工捐赠计划](https://aka.ms/msgive) 做出的贡献：

[@Germey(崔庆才)](https://github.com/Germey), [@SpongeOnline(王创)](https://github.com/SpongeOnline), [@ellie-mac(陈佳佩)](https://github.com/ellie-mac), [@Yawn(刘俊钦)](https://github.com/Aqinqin48), [@White(刘子凡)](https://github.com/jkfhklh), [@597(姜志鹏)](https://github.com/JZP1996)

![Microsoft Give](docs/images/Give_WebBanner.png)

<span id="license-trademarks"></span>
## 许可证和商标

> 整个代码库都在 [MIT许可证](https://github.com/microsoft/HydraLab/blob/main/LICENSE)下。

Reviewer: 也许可以考虑：“整个代码库遵循MIT许可协议。”

Reviewer: ```suggestion
整个代码库遵循 [MIT许可协议](https://github.com/microsoft/HydraLab/blob/main/LICENSE)。
```

> 此项目可能包含项目、产品或服务的商标或徽标。对Microsoft商标或徽标的授权使用受 [Microsoft的商标和品牌准则](https://www.microsoft.com/en-us/legal/intellectualproperty/trademarks/usage/general)的约束，并且必须遵循该准则。在本项目的修改版本中使用Microsoft商标或徽标不得引起混淆或暗示Microsoft的赞助。对第三方商标或徽标的任何使用均受这些第三方政策的约束。

Reviewer: 这样可能更通顺一些：

该项目可能包含一些项目、产品或服务的商标或标识。
使用 Microsoft 商标或标识需遵循 [Microsoft 商标和品牌指导] 并经授权。
在此项目的修改版本中使用 Microsoft 商标或标识不得混淆或暗示 Microsoft 的赞助。
任何使用第三方商标或标识的行为均需遵守相关第三方政策。

Reviewer: ```suggestion
该项目可能包含一些项目、产品或服务的商标或标识。使用 Microsoft 商标或标识需遵循 [Microsoft的商标和品牌准则](https://www.microsoft.com/en-us/legal/intellectualproperty/trademarks/usage/general)并经授权。在此项目的修改版本中使用 Microsoft 商标或标识不得混淆或暗示 Microsoft 的赞助。任何使用第三方商标或标识的行为均需遵守相关第三方政策。
```
